CHANGELOG.rst

```diff
@@ -1,6 +1,12 @@
 Changelog for Mantarray File Manager
 ====================================
 
+0.3.0 (2020-09-17)
```

Comment:
please update buildspec to create a bdist_wheel.  also double check that the .pre-commit-config is up to date with latest hook versions from the python-repo-template (this should be a general practice whenever entering a repo that hasn't been edited in a while)

Reply:
From what I can tell .pre-commit-config is up to date